content/docs/2.10/scalers/kubernetes-workload.md

```diff
@@ -12,24 +12,24 @@ go_file = "kubernetes_workload_scaler"
 triggers:
 - type: kubernetes-workload
   metadata:
-    podSelector: 'app=backend'
-    value: '0.5'
-    activationValue: '3.1'
+    podSelector: app=backend
+    value: 0.5
+    activationValue: 3.1
```

Comment:
this is not correct, yaml parser would defer types as `float` for these values and the `ScaledObject` will be rejected because trigger metadata are `map[string]string`. This has to be explicitly put in quotes to enforce `string` as a type

```suggestion
    value: '0.5'
    activationValue: '3.1'
```